homeassistant/components/knx/weather.py

```diff
@@ -85,13 +93,9 @@ def temperature(self) -> float | None:
 
     @property
     def pressure(self) -> float | None:
-        """Return current air pressure."""
+        """Return current air pressure in pA."""
         # KNX returns pA - HA requires hPa
```

Comment:
Can you please remove the comment here too? It is no longer valid.

Reply:
Oh, you were faster xD